db/schema.rb

```diff
@@ -45,6 +45,74 @@
     t.index ["ancestry"], name: "index_documents_on_ancestry"
   end
 
+  create_table "lit_incomming_localizations", id: :serial, force: :cascade do |t|
```

Comment:
why your schema was changed?

Reply:
Every time I do git pull I also  run 'bundle install' and 'rails db:migrate'.
Probably the last one made some changes to schema.rb.
In any event I did no changes in the db/schema.rb explicitly.